content/en/docs/Responses/structuredLyrics.md

```diff
@@ -71,14 +71,14 @@ Does not exist.
-| ---------- | -------------------------- | ------- | ------- | -------------------------------------------------------------------------------- |
-| `lang`     | `string`                   | **Yes** | **Yes** | The lyrics language                                                              |
-| `synced`   | `boolean`                  | **Yes** | **Yes** | True if the lyrics are synced, false otherwise                                   |
-| `line`     | Array of [`line`](../line) | **Yes** | **Yes** | The actual lyrics. Ordered by start time (synced) or appearance order (unsynced) |
-| `artist`   | `string`                   | No      | **Yes** | The artist name                                                                  |
-| `artistId` | `string`                   | No      | **Yes** | The artist id                                                                    |
-| `title`    | `string`                   | No      | **Yes** | The song name                                                                    |
+| Field           | Type                       | Req.    | OpenS.  | Details                                                                                                                                                                |
+| --------------- | -------------------------- | ------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `lang`          | `string`                   | **Yes** | **Yes** | The lyrics language                                                                                                                                                    |
+| `synced`        | `boolean`                  | **Yes** | **Yes** | True if the lyrics are synced, false otherwise                                                                                                                         |
+| `line`          | Array of [`line`](../line) | **Yes** | **Yes** | The actual lyrics. Ordered by start time (synced) or appearance order (unsynced)                                                                                       |
+| `displayArtist` | `string`                   | No      | **Yes** | The artist name to display. This could be the localized name, or any other value                                                                                       |
+| `displayTitle`  | `string`                   | No      | **Yes** | The title to display. This could be the song title (localized), or any other value                                                                                     |
+| `offset`        | `number`                   | No      | **Yes** | The offset to apply to all lyrics, in milliseconds. Positive means lyrics appear sooner, negative means later. If not included, the offset **must** be assumed to be 0 |
```

Comment:
Why do we need this field? Do lyrics sources sometimes return the timestamps misaligned? If so, could the server pre-process the lyrics and update the `start` timestamp for all the lines to take care of the offset?

Reply:
This is a field of LRC often used to perfectly sync to the user liking and it allows the client to know it's an aligned lyric and so may decide to not apply it's default internal offset in those cases.